setup.py

```diff
@@ -19,6 +19,7 @@
         "pymysql",
         "requests",
         "xlrd",
+        "cryptography",
```

Comment:
I added this in a commit yesterday and made it alphabetical :+1:, so I would remove that commit from this PR.

Reply:
sweet